content/articles/how-to-create-nlp-application-with-flair/index.md

```diff
@@ -0,0 +1,424 @@
+Flair is a natural language processing (NLP) used to build machine learning models used in text classification and speech recognition. It is used in language translation applications, speech recognition, and conversion of speech to text and text to speech applications. It's made up of a text embedding library that enables one to combine different word and document embeddings during model building.
+
+In this tutorial we will start with the Flair basics, then build a simple Natural Language Processing model using flair, the model build is a text classification model that is used to classify a given text as either offensive or not.
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [How to Install Flair](#how-to-install-flair)
+- [Prepare the dataset](#prepare-the-dataset)
+- [Splitting the dataset](#splitting-the-dataset)
+- [Building Corpus](#building-corpus)
+- [Creating Label Dictionary](#creating-label-dictionary)
+- [Word Embeddings with Flair](#word-embeddings-with-flair)
+- [Building and training the model](#building-and-training-the-model)
+- [Making Predictions using our model](#making-predictions-using-our-model)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Prerequisites
+1. You must have a good knowledge of [python](https://www.python.org/) and have it installed in your machine.
+2. You must have a good understanding of machine learning modeling.
+3. You must have a working knowledge of [Pandas](https://numpy.org/).
+4. You must have a working knowledge of [Numpy](https://numpy.org/).
+5. Use [Google Colab](https://research.google.com/) or [Jupyter Notebook](https://jupyter.org/). In this tutorial, we will be using Google Colab in building our model.
+
+### How to install Flair
+Since we are using [Google Colab](https://research.google.com/), use the following command to install flair:
+
+```python
+!pip install flair
+```
+After you have successfully installed Flair we can now start exploring Flair.
+
+### Loading the Flair package
+To load Flair so that we can start using it, we use the following command
+
+```python
+import flair
+```
+
+### Exploring Flair
+Exploring Flair enables us to see the methods and attributes available in this library, these methods will help us in building our text classification model.
+
+```python
+dir(flair)
+```
+
+Output of available methods and attributes:
+
+```bash
+['AnnealOnPlateau',
+ 'Path',
+ '__builtins__',
+ '__cached__',
+ '__doc__',
+ '__file__',
+ '__loader__',
+ '__name__',
+ '__package__',
+ '__path__',
+ '__spec__',
+ '__version__',
+ 'cache_root',
+ 'data',
+ 'datasets',
+ 'device',
+ 'embedding_storage_mode',
+ 'embeddings',
+ 'file_utils',
+ 'logger',
+ 'logging',
+ 'models',
+ 'nn',
+ 'optim',
+ 'os',
+ 'tokenization',
+ 'torch',
+ 'trainers',
+ 'training_utils',
+ 'visual']
+```
+
+The methods shown above such as `embeddings`, `data`, `models`, and `trainers` will be in building our model.
```

Comment:
....will be in building our model?? Did you mean it will be used in building our model? Always proofread our work.